client/src/components/GameContainer.jsx

```diff
@@ -0,0 +1,122 @@
+                    </Box>
+                }
+            </HStack>
+            {hasStatus && !isDesktop &&
+                <Box align="center">
+                    <Tag type={game.status} text={game.status}/>
+                </Box> 
+            }
+            <Divider orientation="horizontal" backgroundColor="gray.500" justifySelf="end" align="top"/>
+
+            {isDesktop ? <Grid templateColumns="3fr 1fr 0.5fr 1fr 3fr" gridGap="1rem" alignItems="center" justifyItems="center" w="100%" fontWeight="semibold" fontSize={isDesktop ? "3rem" : "1.5rem"}>
+                <Text color="greyText.500" fontSize={isDesktop ? "3rem" : "1.5rem"}>{game?.team1?.team?.name}</Text>
+
+                {hasResults ?
+                    <Text justifySelf="end" color="greyText.500"> {game.team1.totalPoints} </Text>
+                    :
+                    <MinusIcon justifySelf="end" color="greyText.500"/>
+                }
+
+                <Divider orientation="vertical" backgroundColor="gray.500" justifySelf="center" align="top" fontSize={isDesktop ? "3rem" : "1.5rem"}/>
+
+                {hasResults ?
+                    <Text justifySelf="start" color="greyText.500"> {game.team2.totalPoints} </Text>
+                    :
+                    <MinusIcon justifySelf="start" color="greyText.500"/>
+                }
+
+                <Text color="greyText.500">{game?.team2?.team?.name}</Text>
+
+            </Grid> 
+            
+            :
+            <VStack>
+                <Grid templateColumns="1fr 1fr" gridGap="1rem" alignItems="center" justifyItems="center" w="100%" fontWeight="semibold" fontSize={isDesktop ? "3rem" : "1.5rem"}>
+                    <Text justifyItems="center" color="greyText.500">{game?.team1?.team?.name}</Text>
+                    <Text justifyItems="center" color="greyText.500">{game?.team2?.team?.name}</Text>
+
+                </Grid>
+                <Grid templateColumns="1fr 1fr" gridGap="1rem" alignItems="center" justifyItems="center" w="100%" fontWeight="semibold" fontSize={isDesktop ? "3rem" : "1.5rem"}>
+                    {hasResults ?
+                        <Text justifyItems="center" color="greyText.500"> {game.team1.totalPoints} </Text>
+                        :
+                        <MinusIcon color="greyText.500"/>
+                    }
+                    {hasResults ?
+                        <Text justifyItems="center" color="greyText.500"> {game.team2.totalPoints} </Text>
+                        :
+                        <MinusIcon color="greyText.500"/>
+                    }
+
+                </Grid>
+            </VStack>
+            }
+            <Divider orientation="horizontal" backgroundColor="gray.500" justifySelf="end" align="top"/>
+
+            <Grid templateColumns="1fr 1fr" gridGap="1rem" alignItems="center" w="100%" justifyItems="center">
+                <HStack>
+                    <TimeIcon color="greyText.500"/>
```

Comment:
[Optional] I actually like this ChakraUI TimeIcon much more than the Icon I used for my date. If you want to do some cleanup, you could probably delete the `DateIcon.jsx` in the svg folder and replace all instances of it with this TimeIcon (`ctrl shift F` to find all `DateIcon` instances, you might need to import the TimeIcon though).